docs/content/guides/developer/first-app/build-test.mdx

```diff
@@ -220,8 +220,10 @@ With the new entry functions included, add another test function to make sure th
             let sword = test_scenario::take_from_sender<Sword>(scenario);
             // verify that the sword has expected properties
             assert!(magic(&sword) == 42 && strength(&sword) == 7, 1);
-            // return the sword to the object pool (it cannot be simply "dropped")
```

Comment:
Dropping the `it cannot be simply "dropped"` because the ability to destroy the sword proves that it can indeed be dropped in a test setting.